src/api/products.ts

```diff
@@ -8,3 +8,7 @@ export const getProducts = async () => {
 export const getProduct = async (id: string) => {
```

Comment:
Try to change the signature to explicitly return a Promise<Product> so if you use this function you know what you get immediately